jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/container/ContainerConfigurationHandler.java

```diff
@@ -0,0 +1,42 @@
+package io.opentelemetry.contrib.jfr.metrics.container;
+
+import io.opentelemetry.api.metrics.Meter;
+import io.opentelemetry.contrib.jfr.Constants;
+import io.opentelemetry.contrib.jfr.metrics.RecordedEventHandler;
+import jdk.jfr.consumer.RecordedEvent;
+
+public class ContainerConfigurationHandler implements RecordedEventHandler {
+    public static final String EVENT_NAME = "jdk.ContainerConfiguration";
+    public static final String JFR_CONTAINER_CONFIGURATION = "jfr.ContainerConfiguration";
+
+    private static final String EFFECTIVE_CPU_COUNT = "effectiveCpuCount";
+
+    private final Meter otelMeter;
+    private volatile long value = 0L;
+
+
+    public ContainerConfigurationHandler(Meter otelMeter) {
+        this.otelMeter = otelMeter;
+    }
+
+    public ContainerConfigurationHandler init() {
```

Comment:
We'd generally set up metrics, etc in the constructor I don't think we need init methods